source/deployment/sso-saml-technical.rst

```diff
@@ -0,0 +1,109 @@
+===================================================
+SAML Single-Sign-On (E20): Technical Documentation
+===================================================
+
+Security Assertion Markup Language (SAML) is an open standard that allows identity providers (IdP), like OneLogin, to pass authorization credentials to service providers (SP), like Mattermost.
+
+In simpler terms, it means you can use one set of credentials to log in to many different sites. With OneLogin account, you can log in to Mattermost, Zendesk, Salesforce and other sites securely with the same account.
```

Comment:
Why do we mention this here? Can we say something like a SAML identity provider instead of mentioning oneLogin?

The reason here is to not allow the question: what if I use OKTA, ADFS or another SAML provider?